fed/mosaic-cell/src/main/java/org/eclipse/mosaic/fed/cell/ambassador/CellAmbassador.java

```diff
@@ -324,27 +328,43 @@ private void process(RsuRegistration rsuRegistration) {
     private void process(TmcRegistration tmcRegistration) {
```

Comment:
Maybe it's possible to have `TmcRegistration`s extend `ServerRegistration`s so the `CellAmbassador` only has to handle `ServerRegistration`s

Reply:
I would prefer to have both registrations separated for now.